lib/Tags/TagsAccordion.js

```diff
@@ -32,6 +32,7 @@ const TagsAccordion = React.memo(({ link, getEntity, getEntityTags, entityTagsPa
             <FormattedNumber value={entityTags.length} />
           </Badge>
         )}
+        closedByDefault={!entityTags.length}
```

Comment:
to be on the safe side I would put `!entityTags.length`
```suggestion
        closedByDefault={!entityTags?.length}
```